PerfettoCds/Pipeline/PerfettoPluginConstants.cs

```diff
@@ -23,6 +23,8 @@ public static class PerfettoPluginConstants
         public const string RawCookerId = "PerfettoRawCooker";
         public const string CounterCookerId = "PerfettoCounterCooker";
         public const string CpuCounterTrackCookerId = "PerfettoCpuCounterTrackCooker";
+        public const string ProcessCounterTrackCookerId = "PerfettoProcessCounterTrackCooker";
```

Comment:
For this, and refactorability, we should use nameof(PerfettoProcessCounterTrackCooker) instead of string so that VS refactor works easily

Reply:
Ya, that's better